week2/project/ecommerce/src/pages/productsList/products/Products.jsx

```diff
@@ -0,0 +1,26 @@
+import './Products.css';
+import Product from './Product'
+
+function Products({ products, isLoading }) {
+  return (
+    <>
+      {isLoading ?
+        <h3>Loading...</h3> :
+        <ul className='products-list'>
+          {products &&
+            products.map(product =>
+                <Product 
+                  key={product.id}  
+                  id={product.id}
+                  image={product.image} 
+                  title={product.title} 
+                  description={product.description} />
+            )
+          }
+        </ul>
```

Comment:
I recommend separating the logic of is loading and showing the products.
for example if you have error isLoading will be false and when this statement try to make a map for products it will crash your app.

✏️need to modify✏️

Reply:
fixed